src/transport/reqwest.rs

```diff
@@ -118,6 +119,11 @@ impl TransportReqwest {
         Self::default()
     }
 
+    /// Use this method to assign custom hostname
+    pub fn set_hostname(&mut self, hostname: &str) {
+        self.hostname = hostname.to_string();
+    }
+
```

Comment:
Do we need that setter here?
`hostname` is now public so it can be changed any time. 

If you want to provide a easy to use setter for many types of data - change the param type to `S: Into<String>`

Reply:
yeah! we can live without setter method here..

Reply:
I think yes - but if you want that method. 
S: Into<String>
is also a valid way to me.